packages/openapi/api/definitions/buckets.yaml

```diff
@@ -0,0 +1,292 @@
+                properties:
+                  status:
+                    type: number
+                    example: 200
+                  result:
+                    type: string
+                    description: Bucket Id.
+        500:
+          $ref: '#/components/responses/internalServerError'
+    delete:
+      operationId: deleteBucket
+      tags:
+        - buckets
+      summary: Deletes a bucket
+      parameters:
+        - name: bucketId
+          in: path
+          description: Bucket to remove.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Bucket successfuly removed.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 200
+        404:
+          $ref: '#/components/responses/bucketNotFound'
+        500:
+          $ref: '#/components/responses/internalServerError'
+  bucketAdlibs:
+    put:
+      operationId: importBucketAdlib
+      tags:
+        - buckets
+      summary: Imports a Bucket Adlib.
+      parameters:
+        - name: bucketId
+          in: path
+          description: Bucket to import the adlib to.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/importAdlib'
+      responses:
+        200:
+          description: Command successfully handled - returns a bucket adlib id.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 200
+                  result:
+                    type: string
+                    description: Bucket Adlib Id.
+        500:
+          $ref: '#/components/responses/internalServerError'
+    delete:
+      operationId: deleteBucketAdlibs
+      tags:
+        - buckets
+      summary: Deletes all adlibs in a bucket
+      parameters:
+        - name: bucketId
+          in: path
+          description: Bucket to remove adlibs from.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Bucket Adlibs successfuly removed.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 200
+        404:
+          $ref: '#/components/responses/bucketNotFound'
+        500:
+          $ref: '#/components/responses/internalServerError'
+  bucketAdlib:
+    delete:
+      operationId: deleteBucketAdlib
+      tags:
+        - buckets
+      summary: Deletes a bucket adlib
+      parameters:
```

Comment:
It looks like this should have an id for the item to be removed?

Reply:
Updated.
Technically the bucketId is not needed because adlibId is already unique, but I'm not sure how the path to the resource should look like then